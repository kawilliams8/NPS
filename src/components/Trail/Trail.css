.Trail {
  color: #989898;
  overflow: scroll;
}

.Trail-image {
  border: 1px solid #989898;
}