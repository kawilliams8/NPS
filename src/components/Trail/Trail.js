import React from 'react';
import '../Trail/Trail.css';

const Trail = ({ trail }) => {
  console.log(trail)
  return (
    <article className="Trail">
      <img src={trail.imgSqSmall} alt={trail.name} className="Trail-image"/>
      <h3>{trail.name}</h3>
      <h4>{trail.length} miles</h4>
      <h4>{trail.summary}</h4>
      <h4>{trail.ascent}</h4>
      <h4>{trail.descent}</h4>
    </article>
  )
}

export default Trail