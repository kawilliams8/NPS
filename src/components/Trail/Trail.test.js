import React from 'react';
import { shallow } from 'enzyme';
import Trail from '../Trail/Trail';

describe('Trail', () => {

  let wrapper, mockTrail;
  beforeEach(() => {
    mockTrail = {
      imgSmall: "image url",
      name: "Mock trail name",
      length: 3,
      ascent: 200,
      descent: 100,
      summary: "Summary"
    }
    wrapper = shallow(<Trail 
    trail={mockTrail}/>)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
}) 