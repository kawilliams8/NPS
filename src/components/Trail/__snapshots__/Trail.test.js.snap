// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trail should match the snapshot 1`] = `
<article
  className="Trail"
>
  <img
    alt="Mock trail name"
    className="Trail-image"
    src="image url"
  />
  <div
    className="Trail-info-wrapper"
  >
    <h3>
      Mock trail name
      , Distance: 
      3
       miles
    </h3>
    <h4>
      Summary
    </h4>
    <h4>
      Ascent: 
      200
       feet, Descent: 
      100
       feet
    </h4>
  </div>
</article>
`;
